qml/SortTypeSelectionDialog.qml

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2019 Open Mobile Platform LLC
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.4
+import Sailfish.Silica 1.0
+import Sailfish.Office.Files 1.0
+
+Dialog {
+
+    property FilterModel filterModel
```

Comment:
This doesn't really need the model for anything else than the sort property. Could consider doing something like signal sortSelected(int) and in the FileList connect that to assigning the value to the actual model.

Reply:
Well, ok. I'll try it

Reply:
Done